workflow.rst

```diff
@@ -509,6 +509,16 @@ missing a title::
 
     The optional second argument of ``setBlocked()`` was introduced in Symfony 5.1.
 
+.. note::
+
+    When using guard listeners which imply intensive workloads (CPU, Database
```

Comment:
This message could be more generic by focusing more on what the announce event do.

I propose to update/add the note in https://symfony.com/doc/current/workflow.html#using-events block (previous one) instead with something like:
```rst
After a transition is applied, the announce event tests for all available transitions.
That will trigger all guard events once more, so you may optimize your workflow
by disabling the announce event if you don't need it.
```